### PR TITLE
mono 5.14.0.177, fsharp 10.0.2

### PR DESCRIPTION
**Pull request by mfilippov**
mono 5.14.0.177, fsharp 10.0.2

### CODE REVIEW

**Comment by mfilippov** [comment]
@BrewTestBot test this please

File: Formula/mono.rb [hunk 1]
```diff
@@ -79,7 +78,6 @@ def install
     if build.with? "fsharp"
       resource("fsharp").stage do
         ENV.prepend_path "PKG_CONFIG_PATH", lib/"pkgconfig"
-        system "./autogen.sh", "--prefix=#{prefix}"
```
Comment (DomT4) [inline Formula/mono.rb]: If we're removing this line can we remove `depends_on "automake" => :build` et al.?

**Comment by DomT4** [comment]
Does anything need doing IRT this?
```
  # When upgrading Mono, make sure to use the revision from
  # https://github.com/mono/mono/blob/mono-#{version}/packaging/MacSDK/msbuild.py
  # NOTE: We're currently using a later revision than from the point release we're using;
  #       the buildsystem changed between them, and 5.12.0.226's is broken.
  #       Revisit this on the next release.
  resource "msbuild" do
    url "https://github.com/mono/msbuild.git",
        :revision => "49a614cda8cedbc6b42e37d49e40cc89fbdac4fd"
  end
```

File: Formula/mono.rb (hunk 1 above)
Comment (mfilippov) [reply Formula/mono.rb]: No, mono depends on automake.

**Comment by mfilippov** [comment]
@DomT4 Thank you for the attention I lost this step.

File: Formula/mono.rb (hunk 1 above)
Comment (DomT4) [reply Formula/mono.rb]: Still? Those dependencies were only added when `fsharp` was back in https://github.com/Homebrew/homebrew-core/commit/949ae5c935b.

Comment (mfilippov) [reply Formula/mono.rb]: Hmm, I see mono in brew just call ./configure without autogen.sh. I'll try to remove it.

**Comment by mfilippov** [comment]
@DomT4 I fixed all issues. PR is ready to merge.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/04c125fe0c8d69447b99f56de8c6029fc756034e. Thank you for your contribution to Homebrew @mfilippov; we appreciate it! :smiley_cat:

**Pull request closed without merging**